resources/carrier/en/30.txt

```diff
@@ -12,9 +12,115 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-30690|Wind
-30693|Wind
+
+# 30 - Greece
+
+# Source(s):
+# -----------------------
+# EETT: http://www.eett.gr/opencms/opencms/EETT_EN/Electronic_Communications/Telecoms/Numbering/search.html?cat=esa_non
+# -----------------------
+
+# Carriers:
+# -----------------------
+# COSMOTE: cosmote.gr - COSMOTE MOBILE TELECOMMUNICATIONS S.A.
+# Vodafone: vodafone.gr - Vodafone-Panafon Hellenic Telecommunications Company S.A.
+# WIND: wind.gr - WIND HELLAS TELECOMMUNICATIONS S.A.
+# Cyta: cyta.gr - Cyta Hellas S.A.
+# Inter Telecom: intertelecom.gr - Inter Telecom O.E.
+# Nova: nova.gr - Forthnet S.A.
+# OTEGLOBE: oteglobe.gr - OTEGLOBE S.A.
+# OTE: cosmote.gr/otegroup - OTE S.A.
+# OSE: ose.gr - OSE S.A.
+# BWS: marketing.gr - Business Web Solutions Ε.Π.Ε.
+# AMD Telecom: amdtelecom.net - AMD Telecom S.A.
+# M-STAT: m-stat.gr - M-STAT S.A.
+# Apifon: apifon.com - Apifon S.M.P.C.
+# Yuboto: yuboto.com - Yuboto Ltd.
+# Compatel: compatel.com - Compatel Ltd.
+# MI Carrier Services: [http://www.solidinfo.se/foretag/mi-carrier-services-ab] MI CARRIER SERVICES AB
+# Premium Net International: premiumnetinternational.com - SC PREMIUM NET INTERNATIONAL SRL
+# -----------------------
+
+# -----------------------
+# Last checked: 7/12/2017
+# Last updated: 7/12/2017
```

Comment:
Please remove all the comments (about carriers, last checked/updated) as it is not needed.